packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -299,10 +299,17 @@ export interface ClusterProps {
   readonly masterUser: Login;
 
   /**
-   * A list of AWS Identity and Access Management (IAM) role that can be used by the cluster to access other AWS services.
+   * A default AWS Identity and Access Management (IAM) role to be used by the cluster to access other AWS services.
+   *
+   * @default - Create a new role
+   */
+  readonly serviceRole?: iam.IRole;
+
+  /**
+   * A list of additional AWS Identity and Access Management (IAM) roles that can be used by the cluster to access other AWS services.
    * The maximum number of roles to attach to a cluster is subject to a quota.
```

Comment:
```suggestion
   * A list of AWS Identity and Access Management (IAM) roles that can be used by the cluster to access other AWS services.
   *
   * The maximum number of roles to attach to a cluster is subject to a quota.
   * @see https://docs.aws.amazon.com/redshift/latest/mgmt/amazon-redshift-limits.html
```
Additional information on quotas.